main/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    description = models.TextField(max_length=300)
+    date_of_birth = models.DateField()
+    timestamp = models.DateTimeField(auto_now_add=True, db_index=True)
+    profile_photo = models.ImageField(null=True, blank=True)
+
+    class Meta:
+        verbose_name = 'User'
+        verbose_name_plural = 'Users'
+
+    def __str__(self):
+        return self.user.username
+
+
+class Post(models.Model):
+    photo = models.ImageField()
+    description = models.TextField(max_length=500, blank=True)
+    timestamp = models.DateTimeField(auto_now_add=True, db_index=True)
```

Comment:
Why do we need it to be an index?

Reply:
Actually I don't know, why I thought it can be useful, remowed